### PR TITLE
Transaction History

### PR DESCRIPTION
**Pull request by amichard**
#428 #429 

This updates the history so it covers more status changes, like if it was rescinded or refused.

Changelog:
- Renamed get_reviewed to get_history
- get_history now returns a list from a single object
- Signatures are now used for verification instead of a separate call
- Changed the format of the entries

### CODE REVIEW

**Comment by amichard** [comment]
[History | Implement User history in Credit Transfer View Screen for Refused action |3|](https://trello.com/c/ivQ5eHAS/711-history-implement-user-history-in-credit-transfer-view-screen-for-refused-action-3)

**Comment by amichard** [comment]
[History | Implement User history in Credit Transfer View Screen for and Rescinded action |3|](https://trello.com/c/Hpp4QgrN/699-history-implement-user-history-in-credit-transfer-view-screen-for-and-rescinded-action-3)

File: backend/api/serializers/CreditTrade.py [hunk 1]
```diff
@@ -404,8 +404,22 @@ def get_comments(self, obj):
 
         return serializer.data
 
-    def get_reviewed(self, obj):
-        serializer = CreditTradeHistoryReviewedSerializer(obj.reviewed)
+    def get_history(self, obj):
```
Comment (plasticviking) [inline backend/api/serializers/CreditTrade.py]: Could we create a test to cover the new endpoint?

Comment (amichard) [reply backend/api/serializers/CreditTrade.py]: Apparently, there was a test for this already and I just didn't update it. It should now be updated.

**Pull request merged**